models/app.model.js

```diff
@@ -5,14 +5,28 @@ exports.fetchTopics = () => {
 };
 
 exports.fetchArticle = id => {
-  return db
+  const commentCount = db
+    .query("SELECT * FROM comments WHERE article_id = $1", [id])
```

Comment:
So I see entirely why you've done it this way, and if it works - then that's cool. But you could refactor this to be way more efficient in a single request to the database by joining the two tables together.

Reply:
done :)